### PR TITLE
[Wiz] Added Dashboards for all the Data Streams

### PR DESCRIPTION
**Pull request by mohitjha-elastic**
Type of change
- Enhancement

## What does this PR do?

- Add dashboards for all the data streams.
- Set unknown to the event.outcome in the pipeline.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally
Clone integrations repo.
Install the elastic package locally.
Start elastic stack using the elastic package.
Move to integrations/packages/wiz directory.
Run the following command to run tests.

## Automated Test

[test-file.txt](https://github.com/elastic/integrations/files/12857659/test-file.txt)

### CODE REVIEW

**Comment by jamiehynds** [comment]
@mohitjha-elastic for the Audit dashboard, could we replace the 'Total Audit' visualisation with 'Audit Events by User', with a pie-chart, breaking down the audit events by user.id. This is likely more useful than displaying the total number of audit events, and can identify Wiz admins/users who have performed the most actions within the Wiz console.

File: packages/wiz/_dev/build/docs/README.md [hunk 1]
```diff
@@ -1,24 +1,22 @@
 # Wiz
 
-This [Wiz](https://www.wiz.io/) integration enables your security team to continuously prioritize critical risks based on a deep cloud analysis across misconfigurations, network exposure, secrets, vulnerabilities, malware, sensitive data and identities to build a single prioritized risk view for your cloud.
-
-Use the Wiz integration to collect and parse data from Wiz api.
+Wiz continuously prioritizes critical risks based on a deep cloud analysis across misconfigurations, network exposure, secrets, vulnerabilities, malware, and identities to build a single prioritized view of risk for your cloud. This [Wiz](https://www.wiz.io/) integration enables you to consume and analyze Wiz data within Elastic Security, including issues, vulnerability data and audit events, providing you with visibility and context for your cloud environments within Elastic Security.
 
 ## Data streams
 
 The Wiz integration collects three types of data: Audit, Issue and Vulnerability.
```
Comment (jamiehynds) [inline packages/wiz/_dev/build/docs/README.md:7]: Can we remove the audit, issue and vulnerability sentences below as this sentence already covers the fact that we support audit, issue and vulnerability.

File: packages/wiz/changelog.yml [hunk 2]
```diff
@@ -1,6 +1,11 @@
 # newer versions go on top
+- version: "0.2.0"
+  changes:
+    - description: Add dashboards for all the data streams.
+      type: enhancement
+      link: hhttps://github.com/elastic/integrations/pull/8154
```
Comment (kcreddy) [inline packages/wiz/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/8154
```

File: packages/wiz/kibana/dashboard/wiz-be3fd3f0-6358-11ee-9db4-21f79f2e6273.json [hunk 3]
```diff
@@ -0,0 +1,801 @@
...
+                                        "75b52a7c-2709-452b-ae98-db21161c203e": {
+                                            "columnOrder": [
+                                                "5e5e4dd7-888c-467b-8fa8-fbbe7b904ff6"
+                                            ],
+                                            "columns": {
+                                                "5e5e4dd7-888c-467b-8fa8-fbbe7b904ff6": {
+                                                    "customLabel": true,
+                                                    "dataType": "number",
+                                                    "isBucketed": false,
+                                                    "label": "Count of Audit",
```
Comment (kcreddy) [inline packages/wiz/kibana/dashboard/wiz-be3fd3f0-6358-11ee-9db4-21f79f2e6273.json:90]:
```suggestion
                                                    "label": "Count of Audit Events",
```

**Review by kcreddy — APPROVED** [review]
LGTM. We can merge after Jamie's approval.

**Review by jamiehynds — APPROVED** [review]

File: packages/wiz/kibana/dashboard/wiz-927c36f0-6358-11ee-a265-c3569aa0cebf.json [hunk 4]
```diff
@@ -0,0 +1,1210 @@
...
+                                    "language": "kuery",
+                                    "query": ""
+                                }
+                            }
+                        },
+                        "description": "",
+                        "id": "",
+                        "params": {
+                            "fontSize": 12,
+                            "markdown": "Navigation\n\n[Wiz Vulnerability (This page)](#/dashboard/wiz-927c36f0-6358-11ee-a265-c3569aa0cebf)\n\n[Wiz Issue](#/dashboard/wiz-d8f91a20-6363-11ee-a265-c3569aa0cebf)\n\n[Wiz Audit](#/dashboard/wiz-be3fd3f0-6358-11ee-9db4-21f79f2e6273)\n\n[Integration Page](/app/integrations/detail/wiz-0.1.0/overview)\n\nOverview\n\nThis dashboard shows Vulnerability overview related to the Wiz Integration.\n\nThis dashboard is made to provide general statistics and show the detection of ingested vulnerabilities.\n\nIt provides information about vulnerability and vulnerability assets. It also displays the distribution of vulnerabilities according to detection method and status. It also contains details regarding count of vulnerabilities over time.",
```
Comment (piyush-elastic) [inline packages/wiz/kibana/dashboard/wiz-927c36f0-6358-11ee-a265-c3569aa0cebf.json:40]: Can we change the navigation to the latest version ?
`wiz-0.1.0/overview` -> `wiz-0.2.0/overview`

**Review by piyush-elastic — APPROVED** [review]

File: packages/wiz/img/wiz-issue-dashboard.png
Comment (jamiehynds) [inline packages/wiz/img/wiz-issue-dashboard.png:1]: @mohitjha-elastic is Event ID the only field we can use in the Issue Details and Issues over Time? Anything that's more human readable like 'Issue Name' or something along those lines? Can we also adjust 'Issues over Timestamp' to 'Issues over Time' (sounds better). Sorry the last minute adjustment.

Comment (mohitjha-elastic) [reply packages/wiz/img/wiz-issue-dashboard.png:1]: Hey @jamiehynds,
We do not have any other fields like the issue name or others so we have been using the Event ID only. However, the Event ID is the Issue ID but we are using the ECS name in the dashboard, let me know if you want us to replace the name Event ID with Issue ID.

No worries, will adjust `Issues over Timestamp` to `Issue over Time`.

Comment (jamiehynds) [reply packages/wiz/img/wiz-issue-dashboard.png:1]: @mohitjha-elastic ok, if the Event ID is the best we can do we can leave it there for now. Would be great to hyperlink each Event ID so users could go to the Wiz console from the dashboard in order to view more information about the Event, but not sure if that's possible?

We can go ahead and merge, but Wiz may have suggestions during the validation process and we can make adjustments if necessary

Comment (mohitjha-elastic) [reply packages/wiz/img/wiz-issue-dashboard.png:1]: I guess it won't be possible to redirect from the Event ID to the Wiz console.
Also, we have adjusted the `Issues over Timestamp` to `Issue over Time`.

**Pull request merged**